packages/esm-covid-app/src/views/lab-results.component.tsx

```diff
@@ -206,7 +206,16 @@ const CovidLabResults: React.FC<CovidLabWidgetProps> = ({ patientUuid }) => {
         ],
       },
     ],
-    [],
+    [
+      config.formNames.CovidLabResultFormName,
+      config.formNames.CovidLabTestFormName,
+      config.formNames.CovidSampleCollectionFormName,
+      config.obsConcepts.covidLabOrderDate_UUID,
+      config.obsConcepts.covidTestResultConcept_UUID,
+      config.obsConcepts.covidTestStatusConcept_UUID,
+      config.obsConcepts.covidTypeofTestConcept_UUID,
+      t,
```

Comment:
This does look right. i don't think the concepts and forms are something that change regularly and don't need to be triggers to the useMemo